osidb/tests/factories.py

```diff
@@ -126,14 +127,29 @@ class Meta:
             )
         ]
     )
-    meta_attr = factory.Dict({"test": "1"})
+    # valid flaw is expected to have certain meta attributes present
+    meta_attr = factory.LazyAttribute(
+        lambda c: {
+            "bz_id": getattr(c, "bz_id", "12345"),
+            "last_change_time": c.updated_dt
+            if isinstance(c.updated_dt, str)
+            else c.updated_dt.strftime(DATETIME_FMT),
+            "test": "1",
```

Comment:
NIT: This is a minor thing. I believe the field `test` was used as placeholder to see if the factory would create meta_attrs, therefore it can be removed as we have other meta_attr being created by default if you will.

Reply:
Good point. I will remove it